tube/spark/__init__.py

```diff
@@ -1,3 +1,4 @@
+import os
```

Comment:
let's:
1. get rid of flask app
2. add documentation about code structure and logic flow in the README
3. add documentation about system components setup ( spark, hadoop, es, postgres, etc) for development mode and production model
4. more docstrings for class & methods & functions